packages/edit-post/src/components/preferences-modal/test/index.js

```diff
@@ -12,31 +12,31 @@ import { useViewportMatch } from '@wordpress/compose';
 /**
  * Internal dependencies
  */
-import PreferencesModal from '../';
+import EditPostPreferencesModal from '../';
 
 // This allows us to tweak the returned value on each test.
 jest.mock( '@wordpress/data/src/components/use-select', () => jest.fn() );
 jest.mock( '@wordpress/compose/src/hooks/use-viewport-match', () => jest.fn() );
 
-describe( 'PreferencesModal', () => {
+describe( 'EditPostPreferencesModal', () => {
```

Comment:
I'm not sure how useful these snapshots actually are as tests 🤷

Reply:
Yeah, I agree. We should be making explicit assertions rather than blindly comparing snapshots. Or maybe just remove them if they aren't helpful 🤷‍♂️.

Reply:
Same, I think there's a fair few component tests that are pretty much "Does it render" and used the snapshot approach to avoid having to write more detailed usability-focused tests. It'd be nice to follow-up at some point to add React Testing Library-based tests that more closely look at how the component's used (e.g. tabbing on larger viewports vs navigation in narrower ones, etc). For the moment, sticking with the snapshots sounds good to me, though, to avoid doing too much in the one PR.

Reply:
Yeah, I'm thinking the best way to test something like this might be e2e, but best look at that in another PR 😅